minibot/scripts/bot_script.py

```diff
@@ -2,5 +2,4 @@
 import time
```

Comment:
Could you make sure bot_script.py doesn't get pushed to GitHub, add this file to the .gitignore

Reply:
Done!